docs-src/ref-programs.scrbl

```diff
@@ -1996,3 +1996,39 @@ There are convenience methods defined for comparing fixed point numbers:
 @index{fxeq} @reachin{fxeq(x, y)} tests whether @tt{x} is equal to @tt{y}.
 
 @index{fxne} @reachin{fxne(x, y)} tests whether @tt{x} is not equal to @tt{y}.
+
+@subsubsection{@tt{Signed Integers}}
+
+The standard library provides abstractions for dealing with signed integers. The following definitions
```

Comment:
Add a `margin-note` about how this is necessary because some Reach target platforms don't support native integers.

Reply:
Add a new card about compiling this efficiently somehow when they do